docs/index.rst

```diff
@@ -33,6 +33,7 @@ Features
     notebooks/3_diagnostics
     notebooks/4_ensemble_reduction
     notebooks/5_warminglevels
+    notebooks/6_yaml
```

Comment:
I would suggest the "6_config"  name instead ? To me "yaml" is an implementation detail.

Reply:
Done